.github/workflows/stale_repos.yml

```diff
@@ -11,18 +11,30 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: actions/checkout@v3
```

Comment:
v4 is the latest

```suggestion
    - uses: actions/checkout@v4
```

Reply:
Yeah, I saw that but thought someone said the other day that had some conflicts with another action so I left it as-is but fine to update and see what happens.